packages/Webkul/Velocity/src/Resources/views/shop/checkout/cart/index.blade.php

```diff
@@ -206,7 +214,7 @@ class="theme-btn light unset">
                                         </button>
                                     </form>
 
-                                    @if ($item->product->getTypeInstance()->showQuantityBox() === true)
+                                    @if ($productOtherThanDownloadable)
```

Comment:
This issue can easily be resolved by setting the property i.e. `showQuantityBox` to `true` here,
https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Product/src/Type/Downloadable.php